index.css

```diff
@@ -384,3 +384,88 @@ button:disabled {
   align-items: center;
   gap: 15px;
 }
+
+/* Contact form */
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  padding: 120px 24px 0 24px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact .content {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  margin-bottom: 50px;
+  line-height: 44px;
+  text-align: center;
+  align-content: center;
+  color: #172b4d;
+}
+
+input,
+textarea {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  width: 356px;
+  border: none;
+  padding-left: 5px;
+  margin-top: 25px;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
```

Comment:
- [ ] Kindly add border-bottom for every input, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155238704-51d22f8b-afe7-440d-8ae3-cf9c0678ee2a.png) | ![image](https://user-images.githubusercontent.com/35535130/155238777-672e8fc2-18fc-44a9-984d-66e705e42603.png) |